\newcommand{\pinfo}[4]{
  #1 & \multicolumn{2}{l}{#2} \\
     & Role: #4 & Website: \href{http://#3}{#3} \\
 \\
}

%\newcommand{\pinfo}[4]{
%  #1 & Role: #4 & Website: \href{http://#3}{#3} \\
%\multicolumn{3}{}{#2} \\
%}

\section*{Informatics Projects}

\subsection*{Software}

For additional software contributions see \href{http://github.com/cmungall}{github.com/cmungall}

\begin{longtable}{p{1in} p{4in} p{2in}}

    \pinfo{OwlSim}{Cross-species phenotype inference, search and matching tool}{owlsim.org}{Main Developer}
    \pinfo{Exomiser}{Functional annotation of variants in clinical genomes}{monarchinitiative.org/page/exomes}{Co-developer}
    \pinfo{TermGenie}{Web-based ontology term generation system}{termgenie.org}{Manager}
    \pinfo{AmiGO}{Gene functional annotation browser}{amigo.geneontology.org}{Manager}
    \pinfo{Noctua}{Functional annotation and network biology environment}{noctua.berkeleybop.org}{Manager}
    \pinfo{Uberon}{Multi-species organ system ontology}{uberon.org}{Lead Developer}
    \pinfo{Phenopackets}{Phenotype exchange format and tools}{github.com/cmungall/phenopacket-format}{Designer}
    \pinfo{SciGraph}{Graph database system}{github.com/SciGraph}{Manager}
    \pinfo{Dipper}{Python biomedical data extraction pipeline}{github.com/monarch-initiative/dipper}{Manager}
    \pinfo{JBrowse}{Javascript genome browser}{jbrowse.org}{Contributor}
    \pinfo{OBO Edit}{Ontology development environment}{oboedit.org}{Manager}
    \pinfo{Chado}{Generic Model Organism Database schema and information system}{gmod.org/wiki/Chado}{Main Designer}
    \pinfo{BioPerl}{Perl bioinformatics library and software collection}{bioperl.org}{Contributor}

\end{longtable}

\subsection*{Database and resources}

For additional contributions, see publication list

\begin{longtable}{p{1in} p{4in} p{2in}}
    \pinfo{Monarch}{Genotype-phenotype information system for humans and model system}{monarchinitiative.org}{Architect, PI}
    \pinfo{GO}{Resource for functional annotation of genes in all species}{geneontology.org}{Informatics Manager}
    \pinfo{Planteome}{Database of plant genotypes and traits}{planteome.org}{co-PI}
    \pinfo{Phenoscape}{Evolutionary phenotype database}{phenoscape.org}{co-PI}
    \pinfo{SDGIO}{Official UN Environment Program Sustainable Development Goals Ontology}{github.com/SDG-InterfaceOntology}{Consultant}
    \pinfo{OBO Library}{Biomedical ontology repository and infrastructure}{obofoundry.org}{Co-founder and developer}
    \pinfo{FANTOMO}{Ontology for functional annotation of the mammalian genome}{github.com/cmungall/fantom5-ontology}{Developer}
    \pinfo{BgeeDb}{Database for gene expression evolution}{http://bgee.org/}{Ontology Developer}
    \pinfo{GloBI}{Database and infrastructure for aggregating and analyzing species interactions}{globalbioticinteractions.org}{Consultant}
    \pinfo{HPO}{Human phenotype ontology}{human-phenotype-ontology.org}{Developer}
    \pinfo{RO}{Biological relationship type ontology}{github.com/oborel}{Lead Developer}
    \pinfo{SO}{Ontology of sequences and genome features}{sequenceontology.org}{Contributor}
    \pinfo{FlyBase}{Drosophila melanogaster database}{flybase.org}{Genome Re-annotation}
    \pinfo{BDGP}{Berkeley Drosophila Genome Project}{bdgp.org}{Analysis Pipeline}
    \pinfo{ArkDb}{Genome mapping database for farmed and other animal species}{thearkdb.org}{Designer}


\end{longtable}
